src/js/package.json

```diff
@@ -105,7 +105,7 @@
   "tsd": {
     "compilerOptions": {
       "lib": [
-        "ES2018",
+        "ES2017",
```

Comment:
What is the point of this? Are you just setting this to the minimum value that allows the build to succeed?

Reply:
Just making things consistent between typescript and tsd configs.